haxe/ui/layouts/Layout.hx

```diff
@@ -28,19 +28,6 @@ class Layout implements ILayout {
             _component.handlePreReposition();
             repositionChildren();
             _component.handlePostReposition();
-
-            if ((component.autoWidth == true || component.autoHeight == true)) {
```

Comment:
Im nervous about this being removed (rather than moved somehow), autosizing (as per our conversation) is fundamental to haxeui (for native backends too)